src/openforms/js/components/admin/form_design/registrations/zgw/ZGWOptionsForm.js

```diff
@@ -92,6 +93,7 @@ ZGWOptionsForm.propTypes = {
     organisatieRsin: PropTypes.string,
     zaakVertrouwelijkheidaanduiding: PropTypes.string,
     medewerkerRoltype: PropTypes.string,
+    productUrl: PropTypes.string,
```

Comment:
Ah nice catch!

Reply:
Every time I'm doing something with prop types I'm saddened that we're not using Typescript yet.